tasks/ubuntu/xenial.task/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+os_version: xenial
+label: Ubuntu Xenial 14.04
```

Comment:
This should be 16.04

Reply:
Indeed! Good catch.

Reply:
I fixed this [in a follow-up commit](https://github.com/puppetlabs/razor-server/commit/6754ab679839e936dd4072ad1286cae6bc6ac086).